### ISSUE TITLE
pre-generate data cubes for MIRISim

### ISSUE THREAD

**Issue by jaymedina** [ExoCTK/exoctk#169]
our function sossFieldSim generates NIRISS data cubes which are then used to create the output contamination plots available to users on exoctk.stsci.edu.

to expand our C&V tool for MIRI LRS slitless observations, we have to generate similar data cubes. when testing out MIRISim, it took about ~5 mins to generate the data cube from their LRS slitless Jupiter NB walkthroughs. as such, we have to pre-generate our data and host it somewhere where the server that runs exoctk can fetch the data quickly, and produce the plots. 

@hover2pi is there a catalog of most-popular regions of the sky / regions that people will most likely be needing our contamination plots for, so I can tackle those first?

**Comment by hover2pi** [comment]
@jaymedina `sossFieldSim` is coordinate agnostic actually so that shouldn't be necessary. It queries the region of sky around the target and finds all the neighbors in the 2MASS catalog. Then it assumes an effective temperature from the 2MASS color. It then grabs a precomputed spectrum from a set of files with rather coarse resolution, scales it to the apparent flux, and adds them to the final image.

"All we need to do" for the other modes is generate a set of files that have a precomputed trace at a variety of stellar temperatures.

We can meet up to talk about it if you like! I wrote code to make SOSS simulations so I'm pretty familiar with the ins-and-outs. Let me know!

**Comment by jaymedina** [comment]
@hover2pi sure, let's meet up tomorrow, I don't have any meetings, so swing by whenever.

**Comment by exowanderer** [comment]
At the in house meeting 2 weeks ago, we talked about using pandexo/pandeia to generate a stack of sims for NIRISS, NIRCam, and MIRI. 

I looked into this and the output of Pandexo is a png file, from Bokeh, that we can load into python using PIL; or we can grab the bokeh input code for the numpy arrays.

Those pngs are generated from Pandeia realistic 2D images of slope files, which is the input to sossFieldSim.

MIRAGE, awesimsoss, and MIRISim are better simulators; but I don't think we need the sims to be high res, only flux v. position accurate 'enough'. Especially in that the high res sims are complicated and in development.

@jaymedina, @hover2pi: is that enough quality for us? Or should we make table of 2D image templates (i.e. what the CV tool does now?
